### PR TITLE
fix: hide blur box when its bottom

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Please...see <File changed> 🙏🏻🥲

- edit typo
- hide blur when its bottom
  스크린샷 2023-07-04 오전 9 38 18

  스크린샷 2023-07-04 오전 9 38 36

### CODE REVIEW

File: apps/web/src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue [hunk 1]
```diff
@@ -158,17 +159,16 @@ interface Props {
 
 const props = defineProps<Props>();
 
+const templateContainerRef = ref<HTMLElement | null>(null);
+const { arrivedState } = useScroll(templateContainerRef);
+const { bottom: isBottom, top: isTop } = toRefs(arrivedState);
 const state = reactive({
     selectedTemplateName: `${TEMPLATE_TYPE.DEFAULT}-${DASHBOARD_TEMPLATES.monthlyCostSummary.name}`,
```
Comment (skdud4659) [inline apps/web/src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue:166]: question: Should responsiveness be considered in this state?

**Review by skdud4659 — APPROVED** [review]
LGTM!
I left a question!

File: apps/web/src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue:166]: I think it was added due to the variable naming.

Comment (yuda110) [reply apps/web/src/services/dashboards/dashboard-create/modules/DashboardTemplateForm.vue:166]: + When selecting a template, the name also changes.

**Review by WANZARGEN — APPROVED** [review]
LGTM

**Pull request merged**